test/lib/target-frameworks.test.ts

```diff
@@ -22,16 +23,25 @@ test('.Net F# project target framework extracted as expected', async (t) => {
   t.deepEqual(targetFrameworks, ['netcoreapp2.1'], 'targetFramework array is as expected');
 });
 
-/*
+/!*
 ****** csproj ******
-*/
+*!/
 
 test('.Net .csproj simple project target framework extracted as expected', async (t) => {
   const targetFrameworks = await extractTargetFrameworksFromFiles(
     `${__dirname}/../fixtures/dotnet-core-simple-project`,
     'simple-project.csproj');
   t.deepEqual(targetFrameworks, ['netcoreapp1.1', 'netcoreapp1.2'], 'targetFramework array is as expected');
 });
+*/
+
+test('.Net .csproj simple project complex target framework extracted as expected', async (t) => {
```

Comment:
Nitpick: shall we be more explicit to what we mean by complex?

```suggestion
test('.Net .csproj simple project with two target frameworks extracted as expected', async (t) => {
```

Reply:
👍